### PR TITLE
Project Ready for review.

### PR DESCRIPTION
**Pull request by AshtonLeeSeloka**

### CODE REVIEW

File: HabitTracker.AshtonLeeSeloka/DBItem/DBItems.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace DBItem
+{
+	public class DBItems
```
Comment (chrisjamiecarter) [inline HabitTracker.AshtonLeeSeloka/DBItem/DBItems.cs:3]: 🟠 **Naming Conventions**

💡 What is a `DBItems`? Would this not be better described as a `Habit`, `HabitRecord`, `HabitItem`?

File: HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+		/// <returns>Correct date value or null</returns>
+		public string? DateInput(string messageToUser) 
+		{
+			Console.Write(messageToUser);
+			string? date = Console.ReadLine();
+
+			if (date == "0")
+				return null;
+
+			while (!DateTime.TryParseExact(date, "dd-MM-yyyy", new CultureInfo("en-Us"), DateTimeStyles.None, out _))
```
Comment (chrisjamiecarter) [inline HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs:34]: 🟠 **Inconsistent Format**

💡 What is it, `dd-MM-yyyy` (line 34) or `dd-MM-yy` (line 36)?

File: README.md
Comment (chrisjamiecarter) [inline README.md:1]: 🟢 **README File**

⭐

File: HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+			}
+		}
+
+
+		/// <summary>
+		/// Inserts data into the table Database 
+		/// </summary>
+		public void InsertRecord() 
+		{
+			CreateDB();
```
Comment (chrisjamiecarter) [inline HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs:80]: 🟠 **Weird Method Placement**

💡 Why are you only creating the database on Insert? What happens if the user performs another action first, like View Records?

**Review by chrisjamiecarter — CHANGES_REQUESTED** [review]
Hey @AshtonLeeSeloka 👋,

Excellent work on your HabitTracker project submission 🎉!

I have performed a peer review. Review/ignore any comments as you wish.

***

🔴 **Application Crashes**

Steps to reproduce:
- Delete the database.
- Run the application.
- Select `4) View Stored Habits`

image

image

Please revisit to try understand and fix these issues 🔧 , as they relate to the projects requirements and they block my approval.

***

Feel free to reach out if you have any questions or if you'd like to collaborate further on any of these points 🆘 .

I will go ahead and wait for you to make your changes, commit and push (which will then notify me) 😊 !

Thanks,
@chrisjamiecarter 👍

File: HabitTracker.AshtonLeeSeloka/DBItem/DBItems.cs (hunk 1 above)
Comment (AshtonLeeSeloka) [reply HabitTracker.AshtonLeeSeloka/DBItem/DBItems.cs]: HI Chris.

Thanks for the feedback , I have updated this class name to Habit.

File: HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs (hunk 2 above)
Comment (AshtonLeeSeloka) [reply HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs]: I have corrected the format to "dd-MM-yyyy" for all instance.
Thanks :)

File: HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs (hunk 3 above)
Comment (AshtonLeeSeloka) [reply HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs]: I have moved this method call to the start of the program , ensuring that a db is always present to perform crud operations against.

**Review by chrisjamiecarter — APPROVED** [review]
Hey @AshtonLeeSeloka 👋 ,

Great job getting your updates pushed through 💪 !

You now fulfill all the requirements 🥇 

***

I will go ahead and mark as approved, keep up the excellent work on the next projects! 😊

Best regards,
@chrisjamiecarter 👍

**Pull request closed without merging**